src/DistortableCollection.js

```diff
@@ -241,9 +241,21 @@ L.DistortableCollection = L.FeatureGroup.extend({
 
     json.images = json.images.reverse();
     json.avg_cm_per_pixel = this._getAvgCmPerPixel(json.images);
-
     return json;
   },
+
+  downloadJson() {
+    const jsonImages = this.generateExportJson(true).images;
+    // a check to prevent download of empty file
+    if (jsonImages.length) {
+      const encodedFile = 'text/json;charset=utf-8,' + encodeURIComponent(JSON.stringify(jsonImages));
+      const a = document.createElement('a');
+      a.href = 'data:' + encodedFile;
+      const fileName = prompt('Input filename');
```

Comment:
We could add extra guidance text to the prompt here, as discussed, like, `Use this file to recover your map’s saved state. Enter a filename:`